libgit2-sys/lib.rs

```diff
@@ -3435,6 +3442,16 @@ extern "C" {
         version: c_uint,
     ) -> c_int;
 
+    pub fn git_diff_patchid(
+        out: *mut git_oid,
+        diff: *mut git_diff,
+        opts: *mut git_diff_patchid_options,
```

Comment:
`libgit2` doesn't seem to have a clear rule on arguments whether they should be `const *` or not. I would've liked to make `diff` and `opts` `*const`, but `libgit2` didn't made them 'const *' although they doesn't seem to be mutated.

Reply:
We just match `libgit2`, so this should be fine.